### PR TITLE
Fixes Restricted Logins to SAML controlled via environment variable #10436 - applies #10449 to master

### PR DESCRIPTION
**Pull request by snipe**
This just applies PR #10449 on `develop` from @adagioajanes to the `master` branch. I've tried to pick what will work on master and apply it there.

### CODE REVIEW

**Comment by snipe** [comment]
Changed target because I'm a dummy.

**Review by uberbrady — COMMENTED** [review]
looking at this very quickly seems fine.

File: resources/lang/en/admin/settings/general.php [hunk 1]
```diff
@@ -174,7 +174,7 @@
     'saml_idp_metadata_help'    => 'You can specify the IdP metadata using a URL or XML file.',
     'saml_attr_mapping_username' => 'Attribute Mapping - Username',
     'saml_attr_mapping_username_help' => 'NameID will be used if attribute mapping is unspecified or invalid.',
-    'saml_forcelogin_label'     => 'SAML Force Login',
+    'saml_forcelogin_label'     => 'SAML Default Login',
```
Comment (uberbrady) [inline resources/lang/en/admin/settings/general.php:177]: I have a small curiousity about the difference between the `saml_forcelogin_label` versus the `saml_forcelogin`, but not enough to block anything.

File: resources/views/settings/saml.blade.php [hunk 2]
```diff
@@ -39,7 +39,7 @@
                 <div class="box-header with-border">
                     <h2 class="box-title">
                         <i class="fa fa-sign-in"></i> SAML
-                    </h4>
+                    </h2>
```
Comment (uberbrady) [inline resources/views/settings/saml.blade.php:42]: Weird side bug to fix, but, hey, I’m game.

**Review by uberbrady — APPROVED** [review]
This seems pretty simple to me, and a great port of @adagioajanes ‘s awesome work towards develop, back over to master. Looks great!

**Pull request merged**